includes/sanitizers/class-amp-allowed-tags-generated.php

```diff
@@ -12026,7 +12014,7 @@ class AMP_Allowed_Tags_Generated {
 					'extension_spec' => array(
 						'deprecated_allow_duplicates' => true,
```

Comment:
This seems directly related to the need in #3489 to de-duplicate. In reality, we can just omit this field entirely from the generated spec data, as we can just _always_ remove duplicates.